dom.bs

```diff
@@ -3040,6 +3042,39 @@ standards that want to define APIs shared between <a for=/>documents</a> and
  <a for=CharacterData>data</a> is the string and <a for=Node>node document</a> is
  <var>document</var>.
 
+ <li><p>Let <var>isScriptElement</var> be true if <var>parent</var> is an {{HTMLScriptElement}}
+ <a for=/>node</a>; otherwise false.
+
+ <li>
+  <p><a for=set>For each</a> <var>value</var> of <var>nodes</var>:
+
+  <ol>
+   <li>
+    <p>If <var>value</var> is a <a for=/>node</a>, then:
```

Comment:
Yeah you're right, I think I wrote it this way originally to make the steps clearer but now the inner bit is 1 step it might aswell just be pulled out.